src/main/java/tc/oc/pgm/api/PGM.java

```diff
@@ -52,6 +57,12 @@
    */
   MapErrorTracker getMapErrorTracker();
 
+  PrefixRegistry getPrefixRegistry();
+
+  void setDefaultPrefixProvider(PrefixProvider prefixProvider);
+
+  PrefixProvider getDefaultPrefixProvider();
```

Comment:
Why `get/setDefaultPrefixProvider` and not just `get/setPrefixProvider` ? Also, i think they should go inside PrefixRegistry : `PGM.get().getPrefixRegistry().setPrefixProvider(provider)`

Reply:
The plugin loads its config before PGMImpl creates the PrefixRegistry, so this is the only way I could think of to set the PrefixProvider to a ConfigPrefixProvider.